README.md

```diff
@@ -1,11 +1,11 @@
-# flat-manager
+ # flat-manager
 
 flat-manager serves and maintains a Flatpak repository. You point it
 at an ostree repository and it will allow Flatpak clients to install
 apps from the repository over HTTP. Additionally, it has an HTTP API
 that lets you upload new builds and manage the repository.
 
-## Building the server
+## Building the server 
```

Comment:
Unnecessary whitespace change?

Reply:
Indeed.

Reply:
I removed the white space change through squashing.